tools-local/tasks/project.json

```diff
@@ -22,7 +24,7 @@
         "portable-net45+win8+wpa81"
       ]
     },
-    "net45": {
+    "net451": {
```

Comment:
Should this reference `Microsoft.TargetingPack.NETFramework.v4.5.1`? It doesn't really matter given that it is building as-is, but it would be more consistent.